1904-12-30.xml

```diff
@@ -0,0 +1,2634 @@
+                            attentive hearers that he would always retain those ties of friendship
+                            and professional respect, which bound him to Alexandria. He referred to
+                            the exalted position he was about to occupy in the Egyptian Government
+                            service, and stated that he entered upon his new duties in the spirit of
+                            a law student--that aspect of a lawyer's career which would remain with
+                            him to the end of his days, no matter to what eminence he attained--and
+                            quoted the remark of Lord Justice Bowen, one of England's greatest
+                            judges, who was wont to say that he had been a law student for 40 years
+                            and would remain a law student as long as he lived.</p>
+                        <p>Maitre Caloyanni then added his testimony to the general ability and
+                            courtesy of his learned friend and late colleague, and proceeded to draw
+                            an interesting comparison between the procedure and methods of English
+                            jurisprudence and that of other countries, where the Code Napoleon and
+                            Roman Law had been adopted.</p>
+                        <p>Mr. Preston followed with a humorous speech, and confirmed what Judge
+                            Long had said as to the invariable sympathy and ungrudging help he had
+                            received at the hands of his colleagues when starting in practice in
+                            Egypt, referring particularly to the kindness and geniality of Mr.
+                            Briscoe, whose health he proposed.</p>
+                        <p>Mr. Briscoe replied in fitting terms. The party did not disperse until a
+                            late hour, after a very pleasant and interesting evening.</p>
+                    </div>
+
+                </div>
+                <div type="section" feature="khedive">
+                    <head>THE KHEDIVE.</head>
+                    <p> H. H. the Khedive, after presiding at the Council of Ministers at Abdeen
+                        Palace on Thursday morning, gave private audiences to Sir William Garstin
+                        and to Lewa Lord Edward Cecil Pasha, and later in the day to the Sheikh El
+                        Jisr of Tripoli (Syria) and to Moh. Bey Beheri, editor of a Moslem newspaper
+                        published at Tripoli.</p>
+                    <p>His Highness will preside over the ceremony of the transport of the Kisweh el
+                        Mahmal from the Citadel to the Mosque of Saidna Hussein.</p>
+                </div>
+
+                <cb n="4"/>
+                <div type="section">
+                    <head>WRECKED PORTUGUESE TRANSPORT.</head>
+                    <div type="item">
```

Comment:
same thing here--don't use the section--just have a series of separate items in this column.